spec/controllers/checkouts_controller_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+require 'stripe_mock'
+
+RSpec.describe CheckoutsController, type: :controller do
+
+  let(:stripe_helper) { StripeMock.create_test_helper }
+  before { StripeMock.start }
+  after { StripeMock.stop }
+
+  describe "GET#new" do
+    before :each do
+      user = create(:user)
+      sign_in user
+      @shop = create(:shop, :user => user)
+    end
+
+    it "goes to the checkout page" do
+      product = create(:product, shop: @shop)
+      get :new, id: product.id
+      cards = create(:card, :user => subject.current_user)
```

Comment:
@LevinGeorge Why are you creating a card here? You don't need to create a card to go to the checkout page.